src/styles/colors.js

```diff
@@ -0,0 +1,10 @@
+export default {
+  debug: 'gold',
```

Comment:
⚪️ I think we should only use hex and not use strings, even though many of these are just temporary.
This site is pretty cool for searching and getting information about specific colors: https://www.colorhexa.com/